src/pybind/mgr/influx/module.py

```diff
@@ -112,13 +113,15 @@ def send_to_influx(self):
         password = self.get_config("password", default="")
 
         client = InfluxDBClient(host, port, username, password, database)
-        databases_avail = client.get_list_database()
-        if database not in databases_avail:
-            self.log.info("Creating database '{0}'".format(database))
-            client.create_database(database)
 
-        client.write_points(self.get_df_stats(), 'ms')
-        client.write_points(self.get_daemon_stats(), 'ms')
+	# using influx client get_list_database requires admin privs, instead we'll catch the not found exception and inform the user if db can't be created
+	try:
+        	client.write_points(self.get_df_stats(), 'ms')
+        	client.write_points(self.get_daemon_stats(), 'ms')
+	except InfluxDBClientError as e:
+                if e.code == 404:
+                        self.log.info("Database '{0}' not found, trying to create (requires admin privs).  You can also create manually and grant write privs to user '{1}'".format(database,username))
+                        client.create_database(database)
```

Comment:
Should also do this:
```
else:
    raise
```

So that if the error is anything other than 404, we're not silently dropping it.